CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ else()
   set(yaml-cpp-label-postfix "static")
 endif()
 
+set(CMAKE_COMPILE_PDB_OUTPUT_DIRECTORY "${CMAKE_INSTALL_LIBDIR}/../")
```

Comment:
It strikes me as a little weird that you're using a parent directory here. Is there a more direct way to put this file in the right spot?